### PR TITLE
docs: add authing blog

### LINKED ISSUE #833: docs: use Canary Release instead of Grayscale
**Actual Behaviour**

Hi, in our posts, we use `Grayscale` to refer to `灰度发布`. e.g https://apisix.apache.org/blog/2021/10/09/apisix-ingress-techblog/

**Expected Behaviour**

Actually, it should be `Canary Release`. We need to replace any text under `Blog`[1] with `grayscale` to `canary release` :)

[1] https://github.com/apache/apisix-website/tree/master/website/blog

### PR DESCRIPTION
**Pull request by yzeng25**
Changes:

- add apisix-authing integration blog
- fix https://github.com/apache/apisix-website/issues/833

### CODE REVIEW

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/01/04/authing.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+4. 在 Authing 控制台中的 「审计日志-用户行为日志」中可以观察到 user1 的登录信息。
+   ![登录信息](https://static.apiseven.com/202108/1641276858261-00fb4d07-751e-42db-bbbf-e8550002466f.png)
+
+## 总结
+
+本文为大家描述了 Apache APISIX 和 Authing 对接的详细操作步骤，通过阅读本文，大家对于在 Apache APISIX 中使用 Authing 有了更清晰的理解。
+
+Apache APISIX 不仅致力于保持自身的高性能，也一直非常重视开源生态的建设。目前 Apache APISIX 已经拥有了 10+ 个认证授权相关的插件，支持与业界主流的认证授权服务对接。
+
+如果你有对接其他认证授权的需求，不妨访问 Apache APISIX 的 [GitHub](https://github.com/apache/apisix/issues)，通过 issue 留下你的建议；或订阅 Apache APISIX 的[邮件列表](https://apisix.apache.org/zh/docs/general/subscribe-guide)，通过邮件表达你的想法。我们也会在每周三晚上 8:00 举行[社区周会](https://meeting.tencent.com/dm/VLjTH3CU3QVi)，欢迎各位参加！
```
Comment (juzhiyuan) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/01/04/authing.md]: > 我们也会在每周三晚上 8:00 举行[社区周会](https://meeting.tencent.com/dm/VLjTH3CU3QVi)，欢迎各位参加！

Hi @yzeng25, we don't have a fixed time for Weekly Meeting :) It happens after someone wants to hold the meeting 😄

**Review by juzhiyuan — DISMISSED** [review]
Others LGTM

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/01/04/authing.md (hunk 1 above)
Comment (yzeng25) [reply website/i18n/zh/docusaurus-plugin-content-blog/2022/01/04/authing.md]: Oh ok, i will modify this part now.

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/01/04/authing.md [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+
+### 步骤一：配置 Authing
+
+1. 登录 Authing 账号，选择自建应用，并填入应用名称和认证地址。如果你没有 Authing 账号，请访问 [Authing 官网](https://www.authing.cn/)，单击右上角 “登录/注册”，注册一个 Authing 账号。
+    ![配置 Authing](https://static.apiseven.com/202108/1641275135400-5a11226a-404d-43f7-bde5-6e1b9599cebd.png)
+
+2. 单击“创建”，创建一个 Authing 应用。
+    ![创建 Authing 应用](https://static.apiseven.com/202108/1641275174810-f78469f7-24dd-41c8-bdc7-eb7d5ff58672.png)
+
+3. 设置登录和登出的跳转 URL。在认证过程中，Authing 将会拒绝除配置以外的回调 URL，由于本次为本地测试，所以将登录回调 URL 和登出回调 URL 都设置为 APISIX 访问地址 http://127.0.0.1:9080/ 。
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/01/04/authing.md:77]:
```suggestion
3. 设置登录和登出的跳转 URL。在认证过程中，Authing 将会拒绝除配置以外的回调 URL，由于本次为本地测试，所以将登录回调 URL 和登出回调 URL 都设置为 APISIX 访问地址 http://127.0.0.1:9080/。
```

Comment (yzeng25) [reply website/i18n/zh/docusaurus-plugin-content-blog/2022/01/04/authing.md:77]: delete this space would result in wrong link` http://127.0.0.1:9080/。`, so i need to keep it as it is now.

**Review by SylviaBABY — DISMISSED** [review]

**Review by takumi913 — DISMISSED** [review]
/LGTM

**Review by juzhiyuan — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**